.github/workflows/crowdin-download.yml

```
@@ -0,0 +1,56 @@
+name: Crowdin download
+
+on:
+  schedule:
+    - cron: '*/15 * * * *'
+  workflow_dispatch:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+      - 'dependabot-**'
```

I'd recommend running this much less frequently (even daily seems too much) and disabling the branch translations because they don't really work.

```suggestion
on:
  schedule:
    - cron: '0 8 * * *'
  workflow_dispatch:
```